modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -101,6 +101,14 @@ $addedItemHighlightColor: #e6ffed;
 	.sheet .panel-group .panel-body {
 		margin-bottom: 0;
 	}
+
+	.border-bottom-2-black {
+		border-bottom: 2px solid #f0f0f0;
+	}
+
+	.min-height-100 {
+		min-height: 100px;
```

Comment:
Hey @jgarcialr, I still think the names of these classes are incorrect. I checked the other ones in this file, and the new ones seem off. 

Could you please take a look at this and apply it to other cases on this pr also?

Thanks

Reply:
I've renamed it to something more meaningful, could you review?